### DISCUSSION TITLE
[Exercise-8]: Write few words about conditional statements and share a sample code.

### DISCUSSION THREAD

**Discussion by acrlakshman** [exaslate-courses/oracle-sql-plsql-b3#9] [General]
* File name: `0-exercices/<your-name>/8-conditionals.sql`

Ref.: TODO

**Comment by cm2024jsp** [comment]
Conditional statements are used to define what logic is to be executed based on the status of some condition being satisfied.
DECLARE
    score NUMBER := 85;
    grade VARCHAR2(1);
BEGIN
    IF score >= 90 THEN
        grade := 'A';
    ELSIF score >= 80 THEN
        grade := 'B';
    ELSIF score >= 70 THEN
        grade := 'C';
    ELSIF score >= 60 THEN
        grade := 'D';
    ELSE
        grade := 'F';
    END IF;

    DBMS_OUTPUT.PUT_LINE('Your grade is ' || grade);
END;
/
